third_party/mbedtls/repo/library/entropy_poll.c

```diff
@@ -50,6 +50,28 @@
 
 #if defined(_WIN32) && !defined(EFIX64) && !defined(EFI32)
 
+#if defined(_KERNEL_MODE)
+
+#include <ntdef.h>
+#include <bcrypt.h>
+
+int mbedtls_platform_entropy_poll( void *data, unsigned char *output, size_t len,
```

Comment:
Is it possible to avoid modifying mbedtls directly by defining a windows-specific `mbedtls_hardware_poll` function?  Alternatively, by defining a windows-specific implementation of `otPlatSecureRandomGet()`?